### PR TITLE
update to `coursier-cache` `1.1.0-M7`

### PR DESCRIPTION
**Pull request by sideeffffect**
force download `jar`s from Nexus proxy, hotfix for https://github.com/scalacenter/scalafix/issues/930

### CODE REVIEW

**Comment by sideeffffect** [comment]
I can't upgrade to `1.1.0-M8` because I need to use `sbt` `0.13` :(

**Review by olafurpg — COMMENTED** [review]
Thanks for this contribution! Have you confirmed that this fixes the problem?

File: small/src/main/scala/com/geirsson/coursiersmall/CoursierSmall.scala [hunk 1]
```diff
@@ -99,8 +99,9 @@ object CoursierSmall {
         (file(cachePolicies.head) /: cachePolicies.tail)(_ orElse file(_)).run
       })
       .unsafeRun()
+      .zip(artifacts.map(_.isOptional))
     val jars = localArtifacts.flatMap {
-      case Left(e) =>
+      case (Left(e), isOptional) if !isOptional =>
```
Comment (alexarchambault) [inline small/src/main/scala/com/geirsson/coursiersmall/CoursierSmall.scala]: Only the `NotFound` errors should be ignored for "optional" artifacts, the others are legit.

Comment (sideeffffect) [reply small/src/main/scala/com/geirsson/coursiersmall/CoursierSmall.scala]: thanks! fixed

**Comment by sideeffffect** [comment]
@olafurpg no, I have not confirmed that this fixes the problem
I'm just implementing what @alexarchambault suggested, but don't really understand how this change could possibly fix the problem...
@alexarchambault could you please confirm that it is reasonable to expect from this change to fix the problem *"coursier doesn't download some jars"*?

**Review by olafurpg — APPROVED** [review]
LGTM 👍 Thank you @sideeffffect! I'm still unsure if this helps the scalafix issue on the other hand

**Pull request merged**